### PR TITLE
fixed a bug when using one register for several middleware

### PR DESCRIPTION
**Pull request by roma-frolov**
# Description

If you use one register for several middleware, then a metric registration conflict occurs.

Error:
```
ValueError: Duplicated timeseries in CollectorRegistry: {'faststream_received_messages_created', 'faststream_received_messages', 'faststream_received_messages_total'}
```

[The error occurs here](https://github.com/airtai/faststream/blob/main/faststream/prometheus/container.py#L50) when trying to register a duplicate metric into a register

## Type of change

- [x] Bug fix (a non-breaking change that resolves an issue)

## Checklist

- [x] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [x] I have conducted a self-review of my own code
- [x] I have made the necessary changes to the documentation
- [x] My changes do not generate any new warnings
- [x] I have added tests to validate the effectiveness of my fix or the functionality of my new feature
- [x] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [x] I have ensured that static analysis tests are passing by running `scripts/static-analysis.sh`
- [x] I have included code examples to illustrate the modifications

### CODE REVIEW

**Comment by Lancetnik** [comment]
I think, we shouldn't rash with this PR and take a break to discuss the problem

**Comment by Lancetnik** [comment]
@draincoder can you help us here?

File: tests/prometheus/basic.py [hunk 1]
```diff
@@ -202,3 +203,31 @@ def assert_publish_metrics(self, metrics_manager: Any):
                 status="success",
             ),
         ]
+
+    async def test_one_registry_for_some_middlewares(
```
Comment (Lancetnik) [inline tests/prometheus/basic.py:207]: Can we make this test in-memory?

Comment (Lancetnik) [reply tests/prometheus/basic.py:207]: Also, seems like we shouldn't publish a message and/or even create a broker to validate target behavior `assert middleware_1._metrics_container is middleware_2._metrics_container`

Comment (roma-frolov) [reply tests/prometheus/basic.py:207]: done, simplified the test

I didn’t see the point in in memory testing in a simplified test

**Review by Sehat1137 — APPROVED** [review]

**Review by Lancetnik — APPROVED** [review]

**Pull request merged**